tools/SetupFlow/DevHome.SetupFlow/Views/Environments/EnvironmentCreationOptionsView.xaml.cs

```diff
@@ -75,4 +87,23 @@ private void AddAdaptiveCardToUI(RenderedAdaptiveCard adaptiveCardData)
             _log.Error(ex, "Error adding adaptive card UI in EnvironmentCreationOptionsView");
         }
     }
+
+    private void AddUIElementsToGrid(Grid gridRemoveItems, Grid gridToAddItems)
```

Comment:
NIT: This code is duplicated once.  Might need to move to a common location in the future.